guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -47,17 +47,43 @@ Apache user must not have access to the keytab file.
 ----
 [service/HTTP]
 mechs = krb5
-cred_store = keytab:/etc/httpd/conf/http.keytab
+cred_store = keytab:/etc/gssproxy/http.keytab
 cred_store = ccache:/var/lib/gssproxy/clients/krb5cc_%U
 euid = __ID_of_Apache_User__
 ----
+. Set the correct SELinux context for the future keytab entry:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# semanage fcontext -a -s system_u -t krb5_keytab_t '/etc/gssproxy/http\.keytab'
+----
 . Create a keytab entry:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-# KRB5_KTNAME=FILE:/etc/httpd/conf/http.keytab net ads keytab add HTTP -U administrator -d3 -s /etc/net-keytab.conf
-# chown root.{apache-user} /etc/httpd/conf/http.keytab
-# chmod 640 /etc/httpd/conf/http.keytab
+# KRB5_KTNAME=FILE:/etc/gssproxy/http.keytab net ads keytab add HTTP -U administrator -d3 -s /etc/net-keytab.conf
+# chown root.root /etc/gssproxy/http.keytab
+# chmod 600 /etc/gssproxy/http.keytab
+----
+// TO BE CONFIRMED: This step should not be needed
+. Add a *Service Principal Name (SPN)* for your AD server.
+You can use a tool such as `adcli`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# adcli spn --add _My_Service_/{foreman-example-com} _My_Username_
+----
+. Make sure the following line is included in the `/etc/sssd/sssd.conf` file:
```

Comment:
```suggestion
. Configure SSSD to map Group Policy Objects (GPOs) from AD to the `foreman` PAM service by ensuring the following line is included in the `/etc/sssd/sssd.conf` file:
```

I don't know if this wording is technically correct, but my point is to suggest to describe what adding the line does.

Reply:
Also, the IMPORTANT box right under this procedure also tells users to add this config option to sssd.conf. That could be confusing. Do you have any ideas for how to deal with it? I think you could take the entire IMPORTANT box and merge it with this step... would that make sense?